src/features/request/PayrollRequest.js

```diff
@@ -146,11 +121,11 @@ export const PayrollRequestContainer = () => {
                   <input
                     type="text"
                     className="payment-form-field-input, amount"
-                    placeholder="1000xxx"
+                    placeholder="Enter amount"
```

Comment:
![Screenshot 2023-04-24 at 11 11 39](https://user-images.githubusercontent.com/45265127/233898718-dd80c6e0-209c-415d-b11b-f7b3cc43b561.png)
I found there is an error when the amount is over 1 million, pls check this

Reply:
@leonthaibao 
OK let me check. Thanks!